posthog/src/main/java/com/posthog/internal/PostHogFlagsResponse.kt

```diff
@@ -21,4 +21,5 @@ public data class PostHogFlagsResponse(
     val flags: Map<String, FeatureFlag>? = null,
     val quotaLimited: List<String>? = null,
     val requestId: String?,
+    val evaluatedAt: Long?,
```

Comment:
add the `@property` comment docs above, also for requestId just noticed its missing